dbos/dbos.py

```diff
@@ -265,15 +265,15 @@ def __init__(
         dbos_logger.info("Initializing DBOS")
         self.config: ConfigFile = config
         self._launched: bool = False
-        self._sys_db: Optional[SystemDatabase] = None
-        self._app_db: Optional[ApplicationDatabase] = None
+        self._sys_db_field: Optional[SystemDatabase] = None
```

Comment:
Why not just `_sys_db`?

Reply:
I changed the property name to `_sys_db` so I had to change this as well. I could have changed the property name to something like `_sys_db_prop` but I figured most of the use is via the property so that should have the shorter name